editor/editor_file_system.cpp

```diff
@@ -2398,18 +2398,22 @@ void EditorFileSystem::reimport_files(const Vector<String> &p_files) {
 	int from = 0;
 	for (int i = 0; i < reimport_files.size(); i++) {
 		if (groups_to_reimport.has(reimport_files[i].path)) {
+			from = i + 1;
 			continue;
 		}
 
 		if (use_multiple_threads && reimport_files[i].threaded) {
-			if (i + 1 == reimport_files.size() || reimport_files[i + 1].importer != reimport_files[from].importer) {
+			if (i + 1 == reimport_files.size() || reimport_files[i + 1].importer != reimport_files[from].importer || groups_to_reimport.has(reimport_files[i + 1].path)) {
 				if (from - i == 0) {
 					// Single file, do not use threads.
 					pr.step(reimport_files[i].path.get_file(), i);
 					_reimport_file(reimport_files[i].path);
 				} else {
 					Ref<ResourceImporter> importer = ResourceFormatImporter::get_singleton()->get_importer_by_name(reimport_files[from].importer);
-					ERR_CONTINUE(!importer.is_valid());
+					if (!importer.is_valid()) {
+						from = i + 1;
+						ERR_CONTINUE(!importer.is_valid());
```

Comment:
```suggestion
						ERR_CONTINUE(true);
```
If it's intended to always fail. No need to check this twice.

Reply:
It's ok to just pass "true"? The problem is that the error message will be something like `Condition "false" is true. Continuing.` instead of `Condition "!importer.is_valid()" is true. Continuing.`

I don't see anywhere else where only true or false was passed to `ERR_CONTINUE`.

You are right that checking the condition twice is not ideal, but I don't see an easy way to increment the counter and call ERR_CONTINUE to have the right error message.

Reply:
`ERR_CONTINUE_MSG`

Reply:
Of course!! Thanks! Did not known that one.